src/main/webapp/script.js

```diff
@@ -90,7 +83,15 @@ function loadHabitElement(habitJson) {
             console.log("ticking...");
             location.reload();
         }
+    })
 
+    tick.addEventListener("click", () => {
+        if (done) 
```

Comment:
can we simplify this part of the code by calling one "toggle done" function since it is determining the isComplete in the servlet so it don't matter here if done is true or false?

Reply:
+1